tests/generic_config_updater/test_multiasic_scenarios.py

```diff
@@ -0,0 +1,192 @@
+import json
+import logging
+import pytest
+import re
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+logger = logging.getLogger(__name__)
+
+IDF_ISOLATION = [
+    {
+        "op": "add",
+        "path": "/asic0/BGP_DEVICE_GLOBAL/STATE/idf_isolation_state",
+        "value": "isolated_no_export"
+    },
+    {
+        "op": "add",
+        "path": "/asic1/BGP_DEVICE_GLOBAL/STATE/idf_isolation_state",
+        "value": "isolated_withdraw_all"
+    },
+]
+
+IDF_UNISOLATION = [
+    {
+        "op": "add",
+        "path": "/asic0/BGP_DEVICE_GLOBAL/STATE/idf_isolation_state",
+        "value": "unisolated"
+    },
+    {
+        "op": "add",
+        "path": "/asic1/BGP_DEVICE_GLOBAL/STATE/idf_isolation_state",
+        "value": "unisolated"
+    },
+]
+
+LINK_CRC_MITIGATION_REMOVE_TEMPLATE = '[{{"op": "remove", "path": "/asic0/PORTCHANNEL_MEMBER/{}|{}"}}]'
+LINK_CRC_MITIGATION_ADD_TEMPLATE = '[{{"op": "add", "path": "/asic0/PORTCHANNEL_MEMBER/{}|{}", "value": {}}}]'
+
+
+def extract_up_interface(output):
+    # Updated regex pattern to match both (U) and (S) status
+    pattern = re.compile(r"^\s*(\d+)\s+(PortChannel\d+)\s+LACP\(\w+\)\(Up\)\s+(Ethernet\d+)\([US]\)", re.MULTILINE)
+    match = pattern.search(output)
+    if match:
+        return match.group(2), match.group(3)
+    else:
+        return None, None
+
+
+@pytest.fixture(autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname):
+    """
+    Setup/teardown fixture for each multi asic test.
+    rollback to check if it goes back to starting config
+    Args:
+        duthosts: list of DUTs.
+        rand_selected_dut: The fixture returns a randomly selected DuT.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    create_checkpoint(duthost)
+
+    yield
+
+    try:
+        logger.info("Rolled back to original checkpoint")
+        rollback_or_reload(duthost)
+    finally:
+        delete_checkpoint(duthost)
+
+
+def test_check_empty_apply_patch(duthost):
```

Comment:
We have this auto run in conftest.py: https://github.com/sonic-net/sonic-mgmt/blob/18519136da6bef38b1cbb501bc74b63d15911cc4/tests/generic_config_updater/conftest.py#L86-L109

Reply:
in condition yaml file, most of GCU test will be skipped if topo is T2 or multi asic T1. Eventually this should be removed then use you pointed test case to verify, but for now, we need it since T2 or multi asic T1 has different path with pizzabox.